content/outline.md

```diff
@@ -0,0 +1,84 @@
+- Expression tree
+- Solver
+- Model/parameter library
+- Simulation and experiments
+- Misc
+
+### How issues get triaged/resolved
+
+We work in one-month sprints. At the monthly maintainer meeting, we assign tasks to be completed by each maintainer for the following month. We assign issues in decreasing order of priority, using the difficulty to judge how long each issue will take. For example, each maintainer may be assigned one hard issue, or five medium ones, or twenty easy ones, or a mix. Any issues that come up during the month will be triaged at the following triaging meeting.
+
+### Closing issues
+
+In some cases, we need more information about the issue before we can judge the priority level and difficulty (or even understand the root cause and hence fix it). In this case, the issue will be marked with a “needs-reply” tag. If there is no response within 30 days, the issue will be closed.
+
+### Removing assignments
+
+Everyone is expected to give an update on the issues they have been assigned either synchronously at the monthly developer meeting or asynchronously by commenting on the issue before the meeting. If no update is given for two consecutive months then the issue is deemed to have been abandoned and will be assigned to someone else (possibly one month for some high-priority issues).
+
+## Contributing
+
+There are many ways to contribute to setting the direction of the project, beyond just code.
+
+### For experienced contributors/maintainers
+
+Look for unassigned issues labelled “priority:high” or “priority:medium”, and volunteer to take them on (either synchronously at the meeting or asynchronously before)
+
+### For first-time contributors
+
+Look for unassigned issues labelled “difficulty:easy” and volunteer to take them on. Come to the monthly newcomers meeting to discuss issues or for help getting started.
+
+### For sponsors (community work orders)
+
+If you work for a company that uses PyBaMM and would like to sponsor the project to ensure its continuing development, there are several ways you can help:
+
+#### Open donations
+
+Recurring donations without a specific deliverable. Donations are acknowledged on public project pages, and opportunities for interaction with the core development team. See our GitHub sponsor page for details.
+
+#### Community work orders
+
+While the roadmap is mainly decided by the core team (maintainers and steering council), you can donate to have issues moved up in priority via Community Work Orders. The price depends on issue difficulty and current priority. Get in touch to find out more.
+
+#### Feedback
+
+We are experimenting with this process and always looking for ways to improve our workflow. If you have any feedback on this process or ideas for how we can improve, please get in touch either by opening a discussion on GitHub or emailing us at [pybamm@gmail.com](mailto:pybamm@gmail.com).
+
+### Meeting links
+
+- Monthly community meeting:
+- Monthly developer meeting:
```

Comment:
We can maybe add the links here, or do they change every week?

Reply:
Tagging @brosaplanella here to add to this

Reply:
We can either include the link to the Google Calendar event or to the Zoom call.

For the developer meeting they are:

- Google: https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MGtxNnExZWc4ZzFiOGVuOXB0c2ZtdnZ2aTlfMjAyMzA2MTJUMTUwMDAwWiBjXzE1Nzc5MGZhNTQ0Y2JmN2UzZjViNjQ5YzcwMGJhNDA0MDRhZDA0NjM0ZjkzNDBlMjU1ZTZkZWY5OWE4OWVkZGFAZw&tmsrc=c_157790fa544cbf7e3f5b649c700ba40404ad04634f9340e255e6def99a89edda%40group.calendar.google.com&scp=ALL
- Zoom: https://numfocus-org.zoom.us/j/89866852955?pwd=dWVzQnpHbXpaTEZCQS9hMU54ZUxodz09

Reply:
The community meetings are happening a bit ad hoc at the moment, once we have a stablished calendar I will share the event.

Reply:
I added the link and I am resolving this conversation for now, since we can update this anytime